kgforge/specializations/stores/bluebrain_nexus.py

```diff
@@ -244,6 +253,30 @@ def _upload_one(self, path: Path, content_type: str) -> Dict:
 
     # C[R]UD.
 
+    @staticmethod
+    def _local_parse(id_value, version_params):
```

Comment:
maybe be completely explicit about what is parsed?  `_local_urlparse`?

Reply:
Just renamed it :)